quinn-proto/src/connection/mod.rs

```diff
@@ -3596,6 +3602,8 @@ pub enum Event {
     Stream(StreamEvent),
     /// One or more application datagrams have been received
     DatagramReceived,
+    /// One or more application datagrams have been sent
```

Comment:
We should probably document that this is only emitted after the datagram buffer was previously blocked? And/or we should rename it, maybe to `DatagramsUnblocked`?